mica-webapp/src/main/webapp/assets/js/mica-data-access-form.js

```diff
@@ -349,7 +348,7 @@ angular.module('formModule', ['schemaForm', 'hc.marked', 'angularMoment', 'schem
         MicaService.toastSuccess(formMessages.validationSuccess);
       } else {
         // an invalid form can be saved with warning
```

Comment:
saving an incomplete form is not an error (as the comment says).

Reply:
Will correct and release another Minor, my mistake.